src/test/java/com/fasterxml/jackson/core/sym/ByteQuadsCanonicalizerTest.java

```diff
@@ -0,0 +1,19 @@
+package com.fasterxml.jackson.core.sym;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ByteQuadsCanonicalizerTest {
+    @Test
+    @Ignore
+    public void testMultiplyByFourFifths() {
+        for (long i = 0; i <= Integer.MAX_VALUE; i++) {
+            int number = (int) i;
+            int expected = (int) (number * 0.80);
+            int actual = ByteQuadsCanonicalizer.multiplyByFourFifths(number);
```

Comment:
this is not a test we want in the source repo - write a test that tests a few important values.

Reply:
Reply in previous comment